packages/sanity/src/core/releases/tool/components/releaseCTAButtons/ReleaseRevertButton/ReleaseRevertButton.tsx

```diff
@@ -208,22 +212,33 @@ const ConfirmReleaseDialog = ({
   )
 }
 
+// TODO:  This is going to be disabled until we have the proper "releases plus" flag
```

Comment:
Do we have a linear story? Afraid this gets lost and nothing is done when we need to do it...